test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -359,23 +358,18 @@ UCS_TEST_SKIP_COND_P(test_reconfig_asymmetric, basic, has_transport("shm"))
 UCS_TEST_SKIP_COND_P(test_reconfig_asymmetric, request_reset,
                      has_transport("shm"), "PROTO_REQUEST_RESET=y")
 {
-    if (has_transport("ib") && !has_resource(sender(), "rc_verbs")) {
-        UCS_TEST_SKIP_R("No IB devs available, config will be non-wireup");
+    if (has_transport("ib") && !has_p2p_transport()) {
+        UCS_TEST_SKIP_R("No p2p TLs available, config will be non-wireup");

```

Comment:
can unite common code with test_ucp_ep_reconfig::run

Reply:
it is not required for all tests, only request_reset

Reply:
so can we share common code with test_ucp_ep_reconfig::request_reset() ?